src/core/util/lang.js

```diff
@@ -1,5 +1,12 @@
 /* @flow */
 
+/**
+ * unicode letters used for parsing html tags, component names and property paths.
+ * use https://www.w3.org/TR/html53/semantics-scripting.html#potentialcustomelementname
+ * except \u10000-\uEFFFF because of performance problem
```

Comment:
I made a benchmark for this and it turns out that in most modern browsers except Safari, it runs fastest if we include `\u10000-\uEFFFF`:

https://jsperf.com/unicode-regex-test

Reply:
Well, in my computer, `npm test` failed because of timeout when including `\u10000-\uEFFFF`. It passed without the extra characters. I don't know why the performance difference exists, yet.